charts/vault/env/dev/plugin-secrets-omd-reader-values.yaml

```diff
@@ -0,0 +1,36 @@
+vault:
```

Comment:
it might be better to merge plugin-secrets-omd-reader-values.yaml and plugin-secrets-kubernetes-reader-values.yaml to a single file that will have both of the plugins. similar to how vault-single-cluster-values.yaml looks like.